Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -38,6 +38,7 @@
 #include <AzFramework/Process/ProcessWatcher.h>
 
 #include <cmath>
+#include <ISystem.h>
```

Comment:
Noooooo =( we worked so hard to keep legacy completely and totally out of here... =((((( There has to be a decoupled way to handle this.

Reply:
Updated! Added bus to AzFramework instead. See @lumberyard-employee-dm comment/recommendation :)